thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/util/CacheUtils.java

```diff
@@ -86,4 +93,30 @@ public static String buildQuery(JsonObject parameters) {
         parameters.getLong("start"),
         parameters.getLong("end"));
   }
+
+  /**
+   * Convert list of strings to their proper URL hosts. For each
+   * string, it will parse out the proper host for the URI.
+   * Example: "http://localhost:8091" -> "localhost"
+   * @param bootstrapUris
+   * @return
```

Comment:
What is return value?

Reply:
oops, forgot to add that in. It returns a list of strings, which are the hosts (e.g. [ "host1", "host2", "host3", "host4"]).